app/views/users/_unfollow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for(current_user.active_relationships.find_by(followed_id: @user.id),
```

Comment:
Same comment goes here for using the declared helper method :))

Reply:
Ýes :)))